.github/workflows/java-8-builds.yml

```diff
@@ -23,10 +23,14 @@ jobs:
                   cache: gradle
             - name: Grant execute permission for gradlew
               run: chmod +x gradlew
-            - name: Build Skript
-              run: ./gradlew nightlyRelease
-            - name: Run test scripts
+            - name: Build Skript and run test scripts
               run: ./gradlew clean skriptTestJava8
+            - name: Rename file
+              run: mv build/libs/Skript.jar build/libs/Skript-test.jar
+            - name: Build Skript nightly
+              run: ./gradlew clean nightlyRelease
+            - name: Rename file
+              run: mv build/libs/Skript.jar build/libs/Skript-nightly.jar
```

Comment:
I don't think it's good to include both in the output, a user wanting to test a build won't know which to choose.